.travis.yml

```diff
@@ -15,7 +15,22 @@ addons:
     chrome: stable
 install:
     - npm install
-# install synapse prerequisites for end to end tests
-    - sudo apt-get install build-essential python2.7-dev libffi-dev python-pip python-setuptools sqlite3 libssl-dev python-virtualenv libjpeg-dev libxslt1-dev
-script:
-    ./scripts/travis.sh
+    - ./scripts/travis/install-deps.sh
+
+matrix:
```

Comment:
how should one interpret this apparently custom section?

Reply:
Haha, ok. So as mentioned on that page, jobs are indeed run in parallel. While speeding things up, it seems things could easily race?

Reply:
Sorry, got pulled into a meeting. There shouldn't be any races since all the jobs are executed in their own isolated containers, so none of the jobs share anything.